docs/guides/index.mdx

```diff
@@ -48,7 +48,7 @@ The short answer is, _not all of it_. The Qiskit Runtime service software that h
 
 
 ## Qiskit Serverless
-Creating utility-scale quantum applications generally requires a variety of compute resource requirements. Premium users can use Qiskit Serverless (package name `qiskit-serverless`) to easily submit quantum workflows for remote, managed execution. See more information [here](./qiskit-serverless) about how to use this collection of tools.
+Creating utility-scale quantum applications generally requires a variety of compute resource requirements. Premium users can use Qiskit Serverless (package name `qiskit-serverless`) to easily submit quantum workflows for remote, managed execution. See more information [here](./serverless) about how to use this collection of tools.
```

Comment:
I think this can be updated to:

> Creating utility-scale quantum applications generally requires a variety of compute resource requirements. Qiskit Serverless provides a simple interface to run workloads across quantum-classical resources. This includes deploying programs to the IBM Quantum Platform, running workloads remotely, and easy resource management for multi-cloud and quantum-centric supercomputing use cases.
> 
> See more information [here](https://docs.quantum.ibm.com/guides/serverless) about how to use this collection of tools to:
> 
> - Parallelize classical tasks, such as pre-processing and post-processing
> - Persist long-running workloads in the cloud, even if your laptop is turned off
> - Deploy reusable programs in the cloud